dev_setup.sh

```diff
@@ -0,0 +1,7 @@
+export SF_ROOT=$(realpath $(dirname $BASH_SOURCE))
```

Comment:
Is there a way to not need this shell script?

Reply:
Yeah I added it by accident. Removed it now.